CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.13.2] - 2022-03-10
+
+### Changed
+
+- Loosened the Gitpython version requirement range
+
```

Comment:
For posterity, this change was made in #196 .

Reply:
Yes, I find it helpful ([here's](https://github.com/fhightower/ioc-finder/blob/main/CHANGELOG.md) and example of how I do it) - especially if the changelog entry is added *after* the actual change is made.